akovshyk/02_GIT.me

```diff
@@ -0,0 +1,75 @@
+  543  cd kovsh/
```

Comment:
Could you please rename file (extension  should be .md
And add git markup too

Reply:
i have fixed it)